internal/services/systemcentervirtualmachinemanager/system_center_virtual_machine_manager_server_resource.go

```diff
@@ -0,0 +1,248 @@
+package systemcentervirtualmachinemanager
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/extendedlocation/2021-08-15/customlocations"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/vmmservers"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/systemcentervirtualmachinemanager/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+type SystemCenterVirtualMachineManagerServerModel struct {
+	Name              string            `tfschema:"name"`
+	ResourceGroupName string            `tfschema:"resource_group_name"`
+	Location          string            `tfschema:"location"`
+	CustomLocationId  string            `tfschema:"custom_location_id"`
+	Fqdn              string            `tfschema:"fqdn"`
+	Username          string            `tfschema:"username"`
+	Password          string            `tfschema:"password"`
+	Port              int               `tfschema:"port"`
+	Tags              map[string]string `tfschema:"tags"`
+}
+
+var _ sdk.Resource = SystemCenterVirtualMachineManagerServerResource{}
+var _ sdk.ResourceWithUpdate = SystemCenterVirtualMachineManagerServerResource{}
+
+type SystemCenterVirtualMachineManagerServerResource struct{}
+
+func (r SystemCenterVirtualMachineManagerServerResource) ModelObject() interface{} {
+	return &SystemCenterVirtualMachineManagerServerModel{}
+}
+
+func (r SystemCenterVirtualMachineManagerServerResource) IDValidationFunc() pluginsdk.SchemaValidateFunc {
+	return vmmservers.ValidateVMmServerID
+}
+
+func (r SystemCenterVirtualMachineManagerServerResource) ResourceType() string {
+	return "azurerm_system_center_virtual_machine_manager_server"
```

Comment:
According to [MS-DOC](https://learn.microsoft.com/en-us/azure/azure-arc/system-center-virtual-machine-manager/overview), this resource is an Arc-enabled SCVMM, not a native Azure SCVMM resource. If so, shall we rename the resource as below like what `azurerm_arc_kubernetes_cluster` does? And could you please also update the name of the folders and files including resource/test/doc?
```suggestion
	return "azurerm_arc_system_center_virtual_machine_manager_server"
```

Reply:
I will confirming with service team about naming.